db/schema.rb

```diff
@@ -31,16 +31,20 @@
     t.index ["organization_id"], name: "index_repositories_on_organization_id"
   end
 
+  create_table "user_org_tables", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.bigint "organization_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["organization_id"], name: "index_user_org_tables_on_organization_id"
+    t.index ["user_id"], name: "index_user_org_tables_on_user_id"
+  end
+
   create_table "users", force: :cascade do |t|
-    t.string "login"
```

Comment:
Need to keep this because not every GitHub user will have a name, but every GitHub user will have a login handle

Reply:
Fixed in latest commit